.github/actions/setup-maven/action.yaml

```
@@ -0,0 +1,24 @@
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: 'maven'
+
+    - name: Cache Maven packages
+      uses: actions/cache@v3
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('pom.xml', 'apps/backend/pom.xml', 'apps/backend/*/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-m2-
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)